docs/mkdocs/get-started/local-setup.md

```
@@ -2,0 +1,174 @@
+08583148b717   mailhog/mailhog                    "MailHog"                About an hour ago   Up About an hour (healthy)   0.0.0.0:1025->1025/tcp, :::1025->1025/tcp, 0.0.0.0:8025->8025/tcp, :::8025->8025/tcp   mailhog
+```
+
+If any service or application is not running for some reason, you can restart the service with the following command:
+```bash
+./run-services.sh restart <service_name>
+```
+To skip build step, use the `-n` option:
+```bash 
+./run-services.sh restart -n <service_name>
+```
+### Step 3: access the application
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)